api/workAllocation/util.ts

```diff
@@ -70,6 +72,50 @@ export function assignActionsToTasks(tasks: any[], view: any): void {
           task.actions = task.actions || [];
           break;
       }
+      task.dueDate = task.due_date
+      task.taskName = task.name
+      task.caseName = task.case_name
+      task.caseCategory = task.case_category
+      task.assigneeName = getAssigneeName(task, caseworkers);
     }
   }
 }
+
+function getAssigneeName(task: any, caseworkers: Caseworker[]): string {
+  if (task.assignee && caseworkers.some(cw => cw.idamId === task.assignee)) {
+    return caseworkers.filter(cw => cw.idamId === task.assignee)[0].firstName;
```

Comment:
I've added the surname in. However, as the udaytest12 user is not currently returned with the list of caseworkers from the backend this can't be shown at the moment anyway. Regardless, once the data is viewable, the last name should be viewable with the first name